src/tsickle.ts

```diff
@@ -163,19 +162,19 @@ class ClosureRewriter extends Rewriter {
       let jsDoc = this.getJSDoc(fnDecl) || [];
 
       // Copy all the tags other than @param/@return into the new
-      // comment without any change; @param/@return are handled later.
-      // TODO: handle for overloads. indexOf on the array doesn't match instances of the Tag type,
-      // so another matching strategy is needed to avoid dupes. Also, there may be problems if
-      // an annotation doesn't apply to all overloads.
+      // JSDoc without any change; @param/@return are handled specially.
+      // TODO: there may be problems if an annotation doesn't apply to all overloads;
```

Comment:
There's also @template T. But I don't know what we'd do in that case, that seems like it just won't fly?